syntax = "proto3";

package BeeGees.Commands.Responses;

message ShipmentDeliveredResponse {
    string ShipmentId = 1;
    int32 ExtraFees = 2;
    bool Success = 3;
}